script/vendor-mimes

```diff
@@ -23,13 +48,27 @@ data.each do |mime, meta|
   next if extensions.empty?
   mimes[mime] = [] if mimes[mime].nil?
   mimes[mime].concat extensions
+
+  # Extract mime-types with "charset" metadata
+  charset_data[mime] = meta["charset"] if meta.key?("charset")
 end
 
+log_info "Formatting primary hash and writing to file.."
 strlen = mimes.keys.max_by(&:length).length
 output = ""
 output << "# Woah there. Do not edit this file directly.\n"
 output << "# This file is generated automatically by script/vendor-mimes.\n\n"
 mimes = mimes.sort_by { |k,v| k }
 output << mimes.map { |mime,extensions| "#{mime.ljust(strlen)} #{extensions.sort.join(" ")}" }.join("\n")
 
+config = File.expand_path "../lib/jekyll/mime.types", __dir__
 File.write(config, output)
+log_info "Done! See: #{config.inspect.white}"
```

Comment:
For the other file, you have `log_success` when you've completed. Should you use log_success here, too?

<img width="1020" alt="Screen Shot 2022-02-11 at 10 08 05 AM" src="https://user-images.githubusercontent.com/237985/153645717-b1568790-5cf3-4c76-807c-86ea62bb0445.png">

Reply:
It was initially. =)
But the green in the middle kinda felt like it broke the flow for me.. :)

Reply:
:)